.gitlab/pipeline/test.yml

```diff
@@ -196,6 +196,8 @@ test-node-metrics:
     # but still want to have debug assertions.
     RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
```

Comment:
not introduced here, but we should remove this "-Cdebug-assertions=y" since we're using testnet profile for tests